### DISCUSSION TITLE
export on roboform to Keepassxc

### DISCUSSION THREAD

**Discussion by ankejapack** [keepassxreboot/keepassxc#11200] [Q&A]
Dear I can't manage to export on roboform to Keepassxc

So I would like to know what to do when inprtering, I am now working with Linux Mint and want to get rid of roboform
But this doesn't work
My english is not that good

**Comment by droidmonkey** [comment]
https://help.roboform.com/hc/en-us/articles/230425008-How-to-export-your-RoboForm-logins-into-a-CSV-file

**Reply by ankejapack** [reply]
That's right, droid monkey, I had found this one, but to export to Keepas you have to change ellen and I don't get that

**Reply by droidmonkey** [reply]
Not sure what "change ellen" means, but you may have to manipulate the CSV a little to get everything to work right.

**Reply by ankejapack** [reply]
That's right, droid monkey, Yes, that's right, that's why I thought someone might know this
